### PR TITLE
Addresses #4250

### PR DESCRIPTION
**Pull request by renebrandel**
_Issue #, if available:_

_Description of changes:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/pages/cli/graphql/custom-business-logic.mdx [hunk 1]
```diff
@@ -559,6 +559,18 @@ responseMappingTemplate: appsync.MappingTemplate.fromFile(path.join(__dirname, "
 
 > **Note:** the `..` is added to the path because the path is always relative to the `build` folder of the custom resource.
 
+## Add authorization rules to custom queries and mutations
+
+Authorization rules can be applied with the `@auth` directive in the same way as field-level authorization rules. See [Field-level authorization rules](~/cli/graphql/authorization-rules/#field-level-authorization-rules) for details.
+
+In the example below, `myCustomMutation` can only be executed by sign-in customers who are authenticated with IAM:
```
Comment (siegerts) [inline src/pages/cli/graphql/custom-business-logic.mdx:566]:
```suggestion
In the example below, `myCustomMutation` can only be executed by signed-in customers who are authenticated with IAM:
```

**Review by alharris-at — COMMENTED** [review]
One comment around styling, otherwise LGTM.

**Review by rachelnabors — APPROVED** [review]

File: src/pages/cli/graphql/custom-business-logic.mdx [hunk 2]
```diff
@@ -559,6 +559,18 @@ responseMappingTemplate: appsync.MappingTemplate.fromFile(path.join(__dirname, "
...
+
+```graphql
+type Mutation {
+  myCustomMutation(args: String): String @auth(rules: [{ allow: private, provider: iam}])
```
[the remark this answers is not in the document]
Comment (rachelnabors) [reply src/pages/cli/graphql/custom-business-logic.mdx]: Note to future self, we could sort code formatting on commit...

**Review by siegerts — APPROVED** [review]

**Review by alharris-at — APPROVED** [review]

File: src/pages/cli/graphql/custom-business-logic.mdx [hunk 3]
```diff
@@ -559,6 +559,18 @@ responseMappingTemplate: appsync.MappingTemplate.fromFile(path.join(__dirname, "
 
 > **Note:** the `..` is added to the path because the path is always relative to the `build` folder of the custom resource.
 
+## Add authorization rules to custom queries and mutations
+
+Authorization rules can be applied with the `@auth` directive in the same way as field-level authorization rules. See [Field-level authorization rules](~/cli/graphql/authorization-rules/#field-level-authorization-rules) for details.
```
Comment (renebrandel) [inline src/pages/cli/graphql/custom-business-logic.mdx:564]:
```suggestion
Authorization rules can be applied with the `@auth` directive in the same way as field-level authorization rules. See [Field-level authorization rules](/cli/graphql/authorization-rules/#field-level-authorization-rules) for details.
```

**Pull request merged**